src/app/GitExtUtils/GitUI/ControlUtil.cs

```diff
@@ -15,9 +15,14 @@ public static IEnumerable<Control> FindDescendants(this Control control,
         {
             Queue<Control> queue = new();
 
+            if (skip?.Invoke(control) == true)
```

Comment:
handle null I assume?
or "is not false"?
```suggestion
            if (skip?.Invoke(control) is true)
```

Reply:
It should be `is true` (so added if no func), but remove skip